app/assets/stylesheets/base.css

```diff
@@ -0,0 +1,63 @@
+.nav-desktop{
+  background-color: #226;
+  line-height: 80px;
+  padding: 0;
+}
+
+.site-logo{
+  color: white;
+  line-height: 80px;
+}
+
+.nav-desktop .menu-desktop{
+  line-height: 90px;
+  background-color: transparent;
+  margin:20px;
+}
+
+.nav-desktop a {
+display: inline-block;
+}
+
+.menu > li > a {
+  display: inline-block;
+  line-height: 90px;
+  padding-top: 0;
+  padding-bottom: 0;
+  color: white;
+  }
+
+.menu > li > a:hover {
```

Comment:
use sass selectors for this http://sass-lang.com/documentation/file.SASS_REFERENCE.html